src/models/goerg_model.py

```diff
@@ -44,15 +46,30 @@ def __init__(self, data: [ReachPoint]):
         self._fit()
         self._max_reach = self._rho
         if data[0].spends:
-            self._cpi = data[0].spends[0] / data[0].impressions[0]
+            if data[0].impressions[0] < 0.001:
```

Comment:
Should it be considered an error if the Goerg model is called with an impression value that is close to zero?